docker-compose.yml

```diff
@@ -26,3 +26,5 @@ services:
       - traefik.enable=true
       - traefik.backend=busybeaver
       - traefik.frontend.rule=Host:busybeaver.sivji.com
+    ports:
```

Comment:
this was great for local dev, but we'll want to remove for production

I need to think of a way to make it easier to develop locally, right now it's a bit of a mess. Maybe I'll have separate docker files for prod and dev?

Reply:
I want to implement flake8 in all my projects. Should be good now.